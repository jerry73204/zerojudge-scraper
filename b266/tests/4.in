3 3 1
4 2 8
8 0 4
5 6 2
1
