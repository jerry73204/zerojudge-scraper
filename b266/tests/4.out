3 3
5 6 2
8 0 4
4 2 8
